docs/conf.py

```diff
@@ -56,9 +56,9 @@ def setup(sphinx):
 # built documents.
 #
 # The short X.Y version.
-version = '0.2.0'
+version = '0.4.1'
```

Comment:
As this is an actual pyhon file, we could also try to read the version from `../CMakeLists.txt`